### PR TITLE
Show more decimals in Display, if possible

### PR DESCRIPTION
**Pull request by Guthenberg**
_123, -123, _-12, or __-1 format=>
1234, _123, 12.3, _1.2, -123, _-12, or -1.0 format

### CODE REVIEW

**Comment by Guthenberg** [comment]
I would like to see the position more accuratly and show decimals if possible.
I am kindof new to this, but I think it would help some people.

**Comment by thinkyhead** [comment]
Still working on it? It's a good idea if we have some places where values passed to `itostr4sign` might go over +999, but not lower than -999.

**Comment by Guthenberg** [comment]
I had a bug in there. I hope I'll find some time to post a improved version tomorrow.

**Comment by boelle** [comment]
if space on display is a problem maybe consider to drop things that are not so relavant... ie during print drop X and Y coords, they dont make sense anyway

**Comment by Guthenberg** [comment]
Found my mistake: DIGIMOD changed.
@ boelle: but this change is meant to make the x and y - Axis-Position more precise - if possible - without of using any extra space on the display: I often print parts that are less than 100mm in size. Therefore a more precise position would be helpfull ( for example 98.7mm). But If I am Printing very large parts it is usefull tu be able to see up to 9999mm. At the same time I want to see positions in the negative Range (like -1.2 or -999).

**Comment by boelle** [comment]
hehe

what i mean is that the coords change so fast during printing that it at least to me make less sense to have them

during setup etc yes then it makes sense

**Comment by thinkyhead** [comment]
@Guthenberg Can you squash the commits down to 1? Use the following:

```
git rebase -i HEAD~6
git push -f
```

File: Marlin/utility.cpp [hunk 1]
```diff
@@ -108,6 +108,28 @@ void safe_delay(millis_t ms) {
...
     return conv;
   }
+// Convert float to rj string with 1234, _123, 12.3, _1.2, -123, _-12, or -1.0 format
+char *ftostr4sign(const float& fx) { 
+  int x=(int)(fx*10); 
+  int xx = abs(x);
+  int cond0=(x >= 0);
+  int cond1=(xx >= 100);
+  int cond2=(xx >= 1000);
+if (!cond1||(cond0&&!cond2)) {
```
Comment (thinkyhead) [inline Marlin/utility.cpp]: Spacing counts. Please adhere to the style established by the rest of the code.

**Review by thinkyhead — CHANGES_REQUESTED** [review]
See https://github.com/MarlinFirmware/Marlin/wiki/DNE-Coding-Standards

**Comment by Guthenberg** [comment]
You're right formatting matters Must have happened when I copyed my changes to the online editor. I hope my last commit fixes the formatting issues. 
Since I've been editing online I have to figure out how to do the squashing.

**Comment by Guthenberg** [comment]
OK, figured that out, I hope.

**Comment by thinkyhead** [comment]
Superseded by #5385 — Implemented as an optional feature.

**Pull request closed without merging**